### ISSUE TITLE
👎 VANILLA RECIPES Broken

### ISSUE THREAD

**Issue by matsitox** [PluginBugs/Issues-ItemsAdder#2143]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already asked on the **#💬ia-community-help** channel on **Discord** to know if anyone already has a solution for the issue.

### Discord tag (optional)

Rologod#4939

### What happened?

As of version 3.3.0-beta-r5 of ItemsAdder.
All vanilla crafts are trampled by Itemsadder's.
The same goes for the default plugin settings, enabling and disabling the Remove Vanilla Recipes option as well.
With any configuration and addition of ItemsAdder it happens.

In the video that shows you will see that I do not have any addition to ItemsAdder and I do not have any configuration altered.

It even allows you to craft ItemsAdder stuff with fewer resources, as well as overriding many vanilla minecraft crafts as such.

Many users have this same problem and it is super serious.

By the way: I have the latest version of the DefaultPack. R9 Beta.
And clearly the latest version of the R8 plugin.
Although it also happens with the R5 version.

PS: I also don't have any other plugin besides ItemsAdder and its dependencies.

**_Video:_** https://youtu.be/JFlKFq5balk

### Steps to reproduce the issue

1. Install ItemsAdder
2. Install Default Pack.
3. And once installed, we will see what happens with the error.
Vanilla crafts are replaced by ItemsAdder and its additions making them impossible to craft.
We can test this with iron boots, we will see that they are replaced by iron plates. And well, it will also let us craft things from the ItemsAdder in a disorderly way and with fewer resources than what the crafting requests.

### Server version

PaperMC 1.19.2 #292

### ItemsAdder Version

3.3.0 BETA - R8

### ProtocolLib Version

ProtocoLIB 1.19 Version Compilation #605

### LoneLibs Version

LoneLibs 1.0.21

### FULL server log

[02:23:43] [ServerMain/INFO]: Building unoptimized datafixer
[02:23:44] [ServerMain/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[02:23:46] [ServerMain/INFO]: Loaded 7 recipes
[02:23:47] [Server thread/INFO]: Starting minecraft server version 1.19.2
[02:23:47] [Server thread/INFO]: Loading properties
[02:23:47] [Server thread/INFO]: This server is running Paper version git-Paper-298 (MC: 1.19.2) (Implementing API version 1.19.2-R0.1-SNAPSHOT) (Git: 309133e)
[02:23:47] [Server thread/INFO]: Using 4 threads for Netty based IO
[02:23:47] [Server thread/INFO]: Server Ping Player Sample Count: 12
[02:23:47] [Server thread/INFO]: [ChunkTaskScheduler] Chunk system is using 1 I/O threads, 4 worker threads, and gen parallelism of 4 threads
[02:23:47] [Server thread/INFO]: Default game type: SURVIVAL
[02:23:47] [Server thread/INFO]: Generating keypair
[02:23:47] [Server thread/INFO]: Starting Minecraft server on *:25565
[02:23:47] [Server thread/INFO]: Using default channel type
[02:23:47] [Server thread/INFO]: Paper: Using Java compression from Velocity.
[02:23:47] [Server thread/INFO]: Paper: Using Java cipher from Velocity.
[02:23:48] [Server thread/INFO]: [LoneLibs] Loading LoneLibs v1.0.21
[02:23:48] [Server thread/INFO]: [LightAPI] Loading LightAPI vbukkit-5.3.0 (build SNAPSHOT)
[02:23:48] [Server thread/INFO]: <LightAPI>: [INFO] Preparing LightAPI...
[02:23:48] [Server thread/INFO]: <LightAPI>: [INFO] Preparing done!
[02:23:48] [Server thread/INFO]: [ProtocolLib] Loading ProtocolLib v5.0.0-SNAPSHOT-b605
[02:23:48] [Server thread/WARN]: [ProtocolLib] Version (MC: 1.19.2) has not yet been tested! Proceed with caution.
[02:23:49] [Server thread/INFO]: [LibsDisguises] Loading LibsDisguises v10.0.31-SNAPSHOT
[02:23:49] [Server thread/INFO]: [ItemsAdder] Loading ItemsAdder v3.3.0-beta-r8
[02:23:49] [Server thread/INFO]: Server permissions file permissions.yml is empty, ignoring it
[02:23:49] [Server thread/INFO]: [LoneLibs] Enabling LoneLibs v1.0.21
[02:23:49] [Server thread/INFO]: [ProtocolLib] Enabling ProtocolLib v5.0.0-SNAPSHOT-b605
[02:23:49] [Server thread/INFO]: Preparing level "world"
[02:23:49] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[02:23:50] [Server thread/INFO]: Time elapsed: 140 ms
[02:23:50] [Server thread/INFO]: Preparing start region for dimension minecraft:the_nether
[02:23:50] [Server thread/INFO]: Time elapsed: 24 ms
[02:23:50] [Server thread/INFO]: Preparing start region for dimension minecraft:the_end
[02:23:50] [Server thread/INFO]: Time elapsed: 20 ms
[02:23:50] [Server thread/INFO]: [LightAPI] Enabling LightAPI vbukkit-5.3.0 (build SNAPSHOT)
[02:23:50] [Server thread/INFO]: <LightAPI>: [INFO] Initializing LightAPI...
[02:23:50] [Server thread/WARN]: java.lang.RuntimeException: (NoSuchFieldException) e
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.bukkit.internal.handler.craftbukkit.nms.v1_19_R1.VanillaNMSHandler.toRuntimeException(VanillaNMSHandler.java:81)
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.bukkit.internal.handler.craftbukkit.nms.v1_19_R1.VanillaNMSHandler.onInitialization(VanillaNMSHandler.java:187)
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.bukkit.internal.handler.craftbukkit.nms.v1_19_R1.StarlightNMSHandler.onInitialization(StarlightNMSHandler.java:167)
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.bukkit.internal.BukkitPlatformImpl.initialization(BukkitPlatformImpl.java:229)
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.common.LightAPI.initialization(LightAPI.java:88)
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.bukkit.BukkitPlugin.onEnable(BukkitPlugin.java:70)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:542)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugin(CraftServer.java:565)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugins(CraftServer.java:479)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:636)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:422)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:306)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1100)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305)
[02:23:50] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
[02:23:50] [Server thread/WARN]: Caused by: java.lang.NoSuchFieldException: e
[02:23:50] [Server thread/WARN]: 	at java.base/java.lang.Class.getDeclaredField(Class.java:2610)
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.bukkit.internal.handler.craftbukkit.nms.v1_19_R1.VanillaNMSHandler.onInitialization(VanillaNMSHandler.java:175)
[02:23:50] [Server thread/WARN]: 	... 15 more
[02:23:50] [Server thread/WARN]: java.lang.IllegalStateException: Initialization failed! Code: -1
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.common.LightAPI.initialization(LightAPI.java:96)
[02:23:50] [Server thread/WARN]: 	at LightAPI-Bukkit-5.3.0.jar//ru.beykerykt.minecraft.lightapi.bukkit.BukkitPlugin.onEnable(BukkitPlugin.java:70)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:542)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugin(CraftServer.java:565)
[02:23:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugins(CraftServer.java:479)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:636)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:422)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:306)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1100)
[02:23:50] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305)
[02:23:50] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
[02:23:50] [Server thread/INFO]: [LightAPI] Disabling LightAPI vbukkit-5.3.0 (build SNAPSHOT)
[02:23:50] [Server thread/INFO]: <LightAPI>: [INFO] Shutdown LightAPI...
[02:23:50] [Server thread/INFO]: [LibsDisguises] Enabling LibsDisguises v10.0.31-SNAPSHOT
[02:23:50] [Server thread/INFO]: [LibsDisguises] File Name: LibsDisguises.jar
[02:23:50] [Server thread/INFO]: [LibsDisguises] Discovered nms version: v1_19_R1
[02:23:50] [Server thread/INFO]: [LibsDisguises] Jenkins Build: #1134
[02:23:50] [Server thread/INFO]: [LibsDisguises] Build Date: 14/11/2022 14:39
[02:23:50] [Server thread/WARN]: [LibsDisguises] If you own the plugin, place the premium jar downloaded from https://www.spigotmc.org/resources/libs-disguises.32453/ in plugins/LibsDisguises/
[02:23:50] [Server thread/INFO]: [LibsDisguises] You are running the free version, commands limited to non-players and operators. (Console, Command Blocks, Admins)
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config configs/combat.yml is out of date (Or missing)! Now refreshing it!
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config configs/features.yml is out of date (Or missing)! Now refreshing it!
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config configs/sanity.yml is out of date (Or missing)! Now refreshing it!
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config configs/players.yml is out of date (Or missing)! Now refreshing it!
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config configs/nametags.yml is out of date (Or missing)! Now refreshing it!
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config configs/commands.yml is out of date (Or missing)! Now refreshing it!
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config configs/libsdisguises.yml is out of date (Or missing)! Now refreshing it!
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config configs/protocol.yml is out of date (Or missing)! Now refreshing it!
[02:23:52] [Server thread/INFO]: [LibsDisguises] Translations for DISGUISES missing! Saving..
[02:23:52] [Server thread/INFO]: [LibsDisguises] Translations for MESSAGES missing! Saving..
[02:23:52] [Server thread/INFO]: [LibsDisguises] Translations for DISGUISE_OPTIONS missing! Saving..
[02:23:52] [Server thread/INFO]: [LibsDisguises] Translations for DISGUISE_OPTIONS_PARAMETERS missing! Saving..
[02:23:52] [Server thread/INFO]: [LibsDisguises] Saved 174 translations that were not in disguises.yml
[02:23:52] [Server thread/INFO]: [LibsDisguises] Saved 261 translations that were not in messages.yml
[02:23:52] [Server thread/INFO]: [LibsDisguises] Saved 190 translations that were not in disguise_options.yml
[02:23:52] [Server thread/INFO]: [LibsDisguises] Saved 1731 translations that were not in disguise_option_parameters.yml
[02:23:52] [Server thread/INFO]: [LibsDisguises] Loaded custom disguise libraryaddict
[02:23:52] [Server thread/INFO]: [LibsDisguises] Loaded 1 custom disguise
[02:23:52] [Server thread/INFO]: [LibsDisguises] Config is up to date!
[02:23:52] [Server thread/INFO]: [ItemsAdder] Enabling ItemsAdder v3.3.0-beta-r8
[02:23:52] [Server thread/INFO]: [NBTAPI] [NBTAPI] Found Spigot: v1_19_R1! Trying to find NMS support
[02:23:52] [Server thread/INFO]: [NBTAPI] [NBTAPI] NMS support 'MC1_19_R1' loaded!
[02:23:52] [Server thread/INFO]: [NBTAPI] [NBTAPI] Found Gson: class com.google.gson.Gson
[02:23:52] [Server thread/INFO]: [ItemsAdder] [0m[38;5;51m
[38;5;51m  ___  ___        __        __   __   ___  __  [38;5;2m    ItemsAdder[38;5;246m 3.3.0-beta-r8
[38;5;51m|  |  |__   |\/| /__`  /\  |  \ |  \ |__  |__) [38;5;2m    LoneLibs[38;5;246m 1.0.21
[38;5;51m|  |  |___  |  | .__/ /--\ |__/ |__/ |___ |  \ [38;5;8m    Paper git-Paper-298 (MC: 1.19.2)
[38;5;51m                                               [0m
[02:23:53] [Server thread/INFO]: [ItemsAdder] [0m[38;5;11mLightAPI not installed, please install it to see blocks emitting light: [38;5;51mhttps://a.devs.beer/lightapi-new[0m
[02:23:53] [Server thread/INFO]: Running delayed init tasks
[02:23:53] [Server thread/INFO]: [ItemsAdder] [0m[38;5;51m[Pack] Extracting internal contents from .jar[0m
[02:23:53] [Server thread/INFO]: [ItemsAdder] [0m[38;5;10m[Pack] DONE extracting internal contents from .jar[0m
[02:23:53] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;10m[License] Spigot product licensed to: [38;5;51mJuliGame14 (880503)[0m
[02:23:54] [Server thread/INFO]: Done (7.831s)! For help, type "help"
[02:23:54] [Server thread/INFO]: Timings Reset
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;11mLoaded [38;5;51m343[38;5;11m items[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;11mUsed [38;5;51m189[38;5;11m/[38;5;51m188[38;5;11m REAL block IDs[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;11mUsed [38;5;51m83[38;5;11m/[38;5;51m750[38;5;11m REAL_NOTE block IDs[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;11mUsed [38;5;51m22[38;5;11m/[38;5;51m63[38;5;11m REAL_TRANSPARENT block IDs[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;11mUsed [38;5;51m0[38;5;11m/[38;5;51m127[38;5;11m REAL_WIRE block IDs[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;11mUsed [38;5;51m5[38;5;11m/[38;5;51m14[38;5;11m FIRE block IDs[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;11mUsed [38;5;51m599[38;5;11m/[38;5;51m6608[38;5;11m font_images[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;10m[Init] Loaded [38;5;51m29[38;5;11m categories[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;10m[Init] Loaded [38;5;51m343[38;5;10m items and stuff in [38;5;51m2048ms[0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;51m[Pack] Checking resourcepack url... [0m
[02:23:55] [Craft Scheduler Thread - 1 - ItemsAdder/INFO]: [ItemsAdder] [0m[38;5;10m[Pack] Resourcepack URL (self-host): http://127.0.0.1:8163/generated.zip[0m

### ItemsAdder config.yml

```shell
metrics: true
update-checker: true
disable-plugin-loading-messages_DONT_ASK_ME_FOR_SUPPORT_IF_YOU_SET_TRUE__THANKS: false
ignore-server-compatibility-check: false
resource-pack:
  hosting:
    no-host:
      enabled: false
    auto-external-host:
      enabled: false
    self-host:
      enabled: true
      server-ip: 127.0.0.1
      pack-port: 8163
    external-host:
      enabled: false
      url: ''
      skip-url-file-type-check___DONT_ASK_HELP_IF_SET_TRUE: false
  apply-on-join: true
  kick-player-on-decline: false
  kick-player-on-fail: false
  delay-ticks: 1
  notify-other-plugins-interfering: true
  title:
    enabled: true
  protect-player:
    black-screen: true
    hide-hud: true
    cancel-movement: true
    lock-player: true
  command:
    usage-cooldown-seconds: 60
  custom-font:
    enabled: false
    name: ipaexg.ttf
    shift: '[0,0]'
    size: 11.0
    oversample: 2.0
  thin-font:
    enabled: false
  zip:
    protect-file-from-unzip:
      protection_1: true
      protection_2: true
    compress-png-files: true
    compress-json-files: true
    generate-custom-armors-textures:
      vanilla_1_17: true
      optifine: true
    disable-overwrite-leather-armor-layers-textures: false
    fix_transparent_noteblocks: false
    ignore-files-ext:
    - db
    - yml
    - xcf
    - psd
    - html
    - gz
    - rar
    - zip
    - 7z
    contents-folders-priority:
    - vanilla
    - _iainternal
  items:
    CustomModelData-starting-value:
      ALL: 10000
      DIAMOND: 11000
config_files:
  lang: en
  dictionaries-lang: en
  force-sync-loading: false
blocks:
  light-api: true
  fix-liquids-flow: true
  fix-floating-blocks: true
  fix-glitched-blocks:
    enabled: true
    only-new-chunks: false
  cancel-break-wrong-tool:
    enabled: true
    send-actionbar: true
  campfire:
    unlit-on-place: true
    allow-any-item: true
  disable-REAL: false
  disable-REAL_NOTE: false
  disable-REAL_TRANSPARENT: false
  disable-REAL_WIRE: true
  disable-FIRE: false
  convert-vanilla-blocks:
    enabled: false
    only-new-chunks: true
items:
  fix-show-attack-and-damage-as-lore: true
  give_death_location_maps: true
  hide-custom-durability-lore: false
liquids:
  enabled: true
huds:
  enabled: true
loots:
  allow-loots-drop-from-spawners-using-silk-touch: false
mobs:
  disable-MOB-behaviour__DISABLE_AT_YOUR_OWN_RISK: false
chat-preview: false
font_images:
  replace-in-customitems-name-and-lore: true
  chat:
    enabled: true
    doublecheck-permission__disable-for-more-performance: false
    add-hover-suggestion: true
  command:
    enabled: true
    commandblocks: true
    excluded:
    - home
    - sethome
    - tpa
  sign:
    enabled: true
  book:
    enabled: true
  anvil:
    enabled: true
  inventory-title:
    enabled: true
  vault-prefix-suffix:
    enabled: true
    update-every-ticks: 1200
  scoreboard-teams:
    enabled: false
    update-every-ticks: 3600
  player-display-name:
    enabled: true
    update-every-ticks: 1200
effects:
  hide-scoreboard-numbers: false
  text-effects:
    enabled: true
    replace-in-customitems-name-and-lore: true
    chat:
      enabled: true
    sign:
      enabled: true
    book:
      enabled: true
    anvil:
      enabled: true
entities:
  max-furniture-vehicles-per-chunk: 30
  allow-removing-old-furnitures_2_3_11: false
  custom-entitites:
    enabled: true
    emotes: true
    interval-rendering-packets-ticks: 1
    optimized-packets: true
recipes:
  hide-vanilla-unrecognized-recipe-errors: true
  fix-packet-too-large-kick: false
  custom-recipes:
    unlock:
      all-on-join: false
      on-pickup:
        any: false
        ingredients: true
  remove-vanilla-recipes:
    enabled: true
    list:
      DIAMOND_SWORD: false
      IRON_INGOT: false
debug:
  items:
    log-files-loading: false
    log-items-loading: false
    log-items-loading-disabled: false
    log-font-images-loading: false
    log-huds-loading: false
    log-attribute-modifiers-loading: false
    log-warnings: false
    log-loots-loading: false
    log-loots-loading-disabled: false
    log-armors-loading: false
  recipes:
    log-recipes-loading: false
    log-recipes-loading-disabled: false
    log-recipes-removed-vanilla: false
    log-recipes-unknown-item: false
    log-fix-packet-too-large: false
  resource-pack:
    log-extract-default-resources: false
    log-not-extract-default-resources-already-exist: false
  blocks:
    log-save: false
    log-save-on-unload: false
    log-mined-unknown-blocks: true
    log-convert-vanilla-blocks: false
  worlds-populators:
    log-loading: false
    log-loading-disabled: false
    log-loading-no-populators: false
    log-loading-no-world: true
    log-generation: false
  trees-populators:
    log-loading: false
    log-loading-disabled: false
    log-loading-no-populators: false
    log-loading-no-world: true
    log-generation: false
  surface-decorators:
    log-loading: false
    log-loading-disabled: false
    log-loading-no-decorators: false
    log-loading-no-world: true
    log-generation: false
  cave-decorators:
    log-loading: false
    log-loading-disabled: false
    log-loading-no-decorators: false
    log-loading-no-world: true
    log-generation: false
  player-stats:
    log-save: false
  performance:
    loading-ms: true
  huds:
    log-triggers: false
  other:
    log-dictionaries-loading: false
    log-liquids-loading: false
    log-minecraft_lang_overwrite-loading: false
  categories:
    log-total-loaded: true
    log-single-loaded: false
    log-single-loaded-overwrite: false
    log-single-skip: false
    log-icon-failed-load: false
  entities:
    log-files-loading: false
    log-entities-loading: false
    log-entities-loading-disabled: false
```

### Other files, you can drag and drop them here to upload. (optional)

In case you need it, I attach the entire configuration folder of my ItemsAdder:
https://www.mediafire.com/file/797xtulxf3hrs5y/ItemsAdder.rar/file

### Screenshots/Videos (you can drag and drop files or paste links)

![3333](https://user-images.githubusercontent.com/74077483/204200503-b97a5b0a-1e93-4fad-8388-380af062d717.png)

**_Video:_** https://youtu.be/JFlKFq5balk

**Comment by matsitox** [comment]
Information to take into account: IF the user does not have permission to manufacture the item.
When trying to craft a Vanilla Minecraft craft, the error will still occur, but instead of spawning the AI item, it will appear empty and when clicked, it will tell you that you don't have permissions to craft that item, despite being a Vanilla craft.

**Comment by Akiranya** [comment]
This bug has been fixed in 3.3.0-r9. Please try the latest version and see if it works
